tests/conftest.py

```diff
@@ -25,12 +30,50 @@ def pytest_configure():
 
 @pytest.fixture
 def get_data():
-    def _get_data(filename: str, as_posix: bool = False) -> PurePath:
+    def _get_data(filename: str) -> Path:
         path = DATADIR / filename
         if not path.exists():
             raise ValueError(f"Path ({path}) does not exist.")
-        if as_posix:
-            path = PurePosixPath(*path.parts)  # type: ignore
         return path
 
     yield _get_data
+
+
+outputs = set()
+
+
+@pytest.fixture
+def get_output():
```

Comment:
How is this function being used?

Reply:
I created it for one of the tests, but I ended up switching to another approach. I've deleted this function for now.